styles/namemc/catppuccin.user.css

```diff
@@ -277,6 +277,7 @@
       background-color: @mantle !important;
     }
 
+    /* main page server list links */
```

Comment:
I don't think this section is needed in the first place then?

![CleanShot 2024-09-07 at 13 37 35](https://github.com/user-attachments/assets/ec4681ca-09b1-4061-a4ae-fbe300fce586)

It would already be themed by the themed variable shown in that screenshot.

Reply:
AH good catch, i'll get rid of that section shortly